docs/connect.html

```diff
@@ -48,6 +48,8 @@ <h4><a id="connect_running" href="#connect_running">Running Kafka Connect</a></h
         <li><code>bootstrap.servers</code> - List of Kafka servers used to bootstrap connections to Kafka</li>
         <li><code>key.converter</code> - Converter class used to convert between Kafka Connect format and the serialized form that is written to Kafka. This controls the format of the keys in messages written to or read from Kafka, and since this is independent of connectors it allows any connector to work with any serialization format. Examples of common formats include JSON and Avro.</li>
         <li><code>value.converter</code> - Converter class used to convert between Kafka Connect format and the serialized form that is written to Kafka. This controls the format of the values in messages written to or read from Kafka, and since this is independent of connectors it allows any connector to work with any serialization format. Examples of common formats include JSON and Avro.</li>
+        <li><code>plugin.path</code> (default <code>empty</code>) - a list of paths that contain
+            plugins (connectors, converters, transformations). For the purpose of quick starts users will have to add the path that contains the FileStreamSourceConnector and FileStreamSinkConnector packaged in <code>connect-file-"version".jar</code>, because these connectors are not included by default to the <code>CLASSPATH</code> or the <code>plugin.path</code> of the Connect worker.</li>
```

Comment:
Previous list item paragraphs are on a single line, and it's probably useful to keep that formatting style. Also, a few nits:
```suggestion
        <li><code>plugin.path</code> (default <code>empty</code>) - a list of paths that contain Connect plugins (connectors, converters, transformations). Before running quick starts, users must add the relative or absolute path that contains the example FileStreamSourceConnector and FileStreamSinkConnector packaged in <code>connect-file-"version".jar</code>, because these connectors are not included by default to the <code>CLASSPATH</code> or the <code>plugin.path</code> of the Connect worker.</li>
```
I also have a few questions.
1. Should the path include the _JAR file_ or the _directory that contains the JAR file_?
2. Would it be useful if we supplied an example `plugin.path` line to add?

I've built the distribution with this PR branch, and verified that the quickstart running the standalone worker fails if the file connectors are not in the `plugin.path`, but adding `plugin.path=libs/connect-file-<version>.jar` does work as long as the quickstart is run from the installation directory.

Reply:
Thanks for the comment @rhauch 
One note that I have is that I wouldn't recommend adding the relative path even if it works, because it possibly doesn't lead to robust deployments. `plugin.path` works when following symlinks so that can help. 

`plugin.path` works with uber-jars but I wouldn't necessarily consider `connect-file-<version>.jar` an uber jar. I'd recommend the mainstream way which is to add the parent directory of any directories containing connector jars. Let me try to see how an example looks like here, because that's already just a bullet point.